src/node/rpc/memberfrontend.h

```diff
@@ -400,6 +402,61 @@ namespace ccf
 
         return jsonrpc::success(complete_proposal(args.tx, vote.id));
       };
+
+      auto create = [this](RequestArgs& args) {
+        const auto in = args.params.get<CreateNetworkNodeToNode::In>();
+        CreateNetworkNodeToNode::Out result;
+
+        GenesisGenerator g(this->network, args.tx);
+        g.init_values();
+        for (auto& cert : in.member_cert)
+        {
+          g.add_member(cert);
+        }
+
+        ccf::NodeInfoNetwork node_info_network{in.node_info_network.host,
+                                               in.node_info_network.pubhost,
+                                               in.node_info_network.nodeport,
+                                               in.node_info_network.rpcport};
+
+        // Generate quote over node certificate
+        // TODO: https://github.com/microsoft/CCF/issues/59
+        size_t self = g.add_node(
+          {node_info_network, in.node_cert, in.quote, NodeStatus::TRUSTED});
+
+        if (self != 0)
+        {
+          return jsonrpc::error(
+            jsonrpc::StandardErrorCodes::INVALID_PARAMS,
+            fmt::format("My node is was set to {}", self));
+        }
+
+#ifdef GET_QUOTE
+        CodeDigest node_code_id;
+        std::copy_n(
+          std::begin(in.code_digest),
+          CODE_DIGEST_BYTES,
+          std::begin(node_code_id));
+        g.trust_code_id(node_code_id);
+#endif
+
+        // set access whitelists
+        // TODO(#feature): this should be configurable
+        for (const auto& wl : default_whitelists)
+        {
+          g.set_whitelist(wl.first, wl.second);
+        }
+
+        g.set_gov_scripts(
+          lua::Interpreter().invoke<nlohmann::json>(in.gov_script));
+
+        g.create_service(in.network_cert);
+
+        return jsonrpc::success(result);
```

Comment:
This could just be `jsonrpc::success(true)` and `CreateNetworkNodeToNode::Out` does not have to be defined.